components/list/list-item-drag-drop-mixin.js

```diff
@@ -351,6 +356,10 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 		super.firstUpdated(changedProperties);
 	}
 
+	activateDragHandle() {
```

Comment:
This new method enables the consumer to reactivate the drag handle after they've re-rendered the list/list-items.